### DISCUSSION TITLE
init lotus-miner failed

### DISCUSSION THREAD

**Discussion by jzhao2004** [filecoin-project/lotus#3946] [General Q&A] [answered]
2020-09-21T11:14:19.673-0400	ERROR	main	lotus-storage-miner/init.go:246Failed to initialize lotus-miner: creating miner failed:
    main.storageMinerInit

lotus-storage-miner/init.go:516
  - failed to sign message: signing using key 't3uvkzigjstia52f4ivdwipdzrtdfbgolhm6rmwejwdydpmcizi5g3ect6vrunzudjz6in4qmqjipmx7p236pq': key info not found

**Comment by SgtCoin** [comment, marked as answer]
It looks like your miner does not have access or info on your key.
